llvm/lib/Transforms/Coroutines/CoroFrame.cpp

```diff
@@ -123,8 +134,9 @@ class SuspendCrossingInfo {
 
 public:
 #if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)
-  void dump() const;
-  void dump(StringRef Label, BitVector const &BV) const;
+  void dump(const ReversePostOrderTraversal<Function *> &RPOT) const;
```

Comment:
I am against not having a zero-argument `dump` overload. 

One benefit of this `dump()` function not taking any parameter is that you can easily dump stuff in a debugger session. Now I will have to look into how to construct a `ReversePostOrderTraversal`, which may not be possible.

Reply:
I can add this as a second dump, keeping the dump with no args.

Reply:
Done, dump() now does not need args.